lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -794,14 +795,52 @@ public void playAnimation() {
       }
     }
     if (!animationsEnabled()) {
-      setFrame((int) (getSpeed() < 0 ? getMinFrame() : getMaxFrame()));
+      Marker markerForAnimationsDisabled = getMarkerForAnimationsDisabled();
+      if (markerForAnimationsDisabled != null) {
```

Comment:
What should determine if "reduced motion" mode is enabled? Reading the code it seems like the reduced motion marker is always used if present, as opposed to only using it when the app / OS "reduced motion" toggle is enabled.

Reply:
It defaults to [this](https://github.com/airbnb/lottie-android/blob/f41f03a13258987ea89e41a17e6df0d3a84adac2/lottie/src/main/java/com/airbnb/lottie/utils/Utils.java#L255) which can be set if battery saver is on or if you turn off animations in developer settings. It's not an a11y setting on Android afaik though.

Reply:
Oh I see, I missed that this is gated by a `animationsEnabled()` check, which can be configured via `setSystemAnimationsAreEnabled`.

I think we'd probably want the "is reduced motion enabled" check to be configured separately from the "are system animations enabled" check.

Reply:
It doesn't feel quite right to take a specific a11y setting on iOS and use the same name on Android when it maps to different things.

Reply:
@gpeal  In the a11y community, "reduced motion" is the more common term even if it comes from Apple. I think the same is applied to lottie-web as well, so designers can produce one asset to share across platform.

Reply:
> It's not an a11y setting on Android afaik though.
It is available under accessibility then Color and Motion
![Color and Motions settings screen](https://github.com/airbnb/lottie-android/assets/1218420/4935b374-6767-4350-a5ef-aee71b528f67)

Reply:
I think that's reasonable. It's worth noting that this _would_ affect non a11y use cases like battery saver and developer settings but this behavior may still be better if the designer was explicit about it.

Could you:
1. Double check that the utils function I linked to returns 0 for animation scale when this a11y setting is enabled
2. Add documentation for `setSystemAnimationsAreEnabled` in case anybody else calls it so they can learn about this without reading the source?

Reply:
Checked Accessibility setting in the utils function, it returns false when running in the debugger.

<img width="550" alt="Screenshot 2024-01-23 at 1 18 54 PM" src="https://github.com/airbnb/lottie-android/assets/1218420/ba66616d-ac7c-4e49-af5c-02405394924f">

Reply:
> Add documentation for setSystemAnimationsAreEnabled in case anybody else calls it so they can learn about this without reading the source?

I added the document comment, thanks for the suggestion